packages/connector-kakao/CHANGELOG.md

```diff
@@ -0,0 +1,105 @@
+# Change Log
```

Comment:
The "CHANGELOG" is an auto generated file, we can remove it, it will be generated on publishing.